go/login/server/server_test.go

```diff
@@ -0,0 +1,155 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	"encoding/hex"
+	"fmt"
+	"io/ioutil"
+	"net/http/httptest"
+	"testing"
+
+	"../../glome"
+)
+
+type testVector struct {
+	kbp         []byte
+	index       uint8
+	ka          []byte
+	Request     string
+	Response    string
+	ResponseLen uint8
+	authFunc    Authorizer
+}
+
+// ServerKey return correctly formatted Server Private Key
+func (t *testVector) SeverKey() glome.PrivateKey {
+	p, err := glome.PrivateKeyFromSlice(t.kbp)
+	if err != nil {
+		panic(fmt.Sprintf("Glome rejected %v:%#v", t.kbp, err))
+	}
+	return *p
+}
+
+// ClientKey return correctly formatted Client Public Key
+func (t testVector) ClientKey() glome.PublicKey {
+	p, err := glome.PublicKeyFromSlice(t.ka)
+	if err != nil {
+		panic(fmt.Sprintf("Glome rejected %v:%#v", t.ka, err))
+	}
+	return *p
+}
+
+func decodeString(s string) []byte {
+	b, err := hex.DecodeString(s)
+	if err != nil {
+		panic(fmt.Sprintf("Invalid hexadecimal string %v input in test", s))
+	}
+	return b
+}
+
+func auth(user string, hostID string, hostIDType string, action string) (bool, error) {
+	return true, nil
+}
+
+func notAuth(user string, hostID string, hostIDType string, action string) (bool, error) {
+	return false, nil
+}
```

Comment:
```suggestion
func alwaysTrue(user string, hostID string, hostIDType string, action string) (bool, error) {
	return true, nil
}

func alwaysFalse(user string, hostID string, hostIDType string, action string) (bool, error) {
	return false, nil
}
```

Maybe that'd be more obvious?

Reply:
Changed to constantTrue